Import emb

chloeyutianyi opened the pull request.

Fleker commented on cyanobyte/codegen.py:
```diff
@@ -21,14 +21,21 @@
 
 _VERSION = "0.1.0"
-_DEBUG = False
+_DEBUG = True
```
Can we keep this as `False`. It can be set using the `--debug true` on the command line

Fleker commented on cyanobyte/codegen.py:
```diff
@@ -85,11 +116,16 @@ def generate_source_file(template, peripheral, opts, template_ext, out_dir):
+                if _DEBUG:
+                    print('Imported ' + emboss_filename)
+                """
                 with open(os.path.join(dir_path, emboss_filename)) as emboss_file:
```
Can we remove the commented lines, as we'll not need them.

Fleker commented on cyanobyte_float.emb.h:
```diff
@@ -0,0 +1,1415 @@
+// Generated by the Emboss compiler.  DO NOT EDIT!
```
We can delete this file from this commit.

Fleker commented on emboss_cyanobyte.py:
```diff
@@ -0,0 +1,86 @@
+"""Main driver program for using cyanobyte as the backend for emboss"""
```
We can delete this file from this commit.

Fleker commented on emboss_float.emb.h:
```diff
@@ -0,0 +1,1460 @@
+// Generated by the Emboss compiler.  DO NOT EDIT!
+#ifndef TESTDATA_FLOAT_EMB_H_
```
We can delete this file.

Fleker commented on emboss_float.yaml:
```diff
@@ -0,0 +1,45 @@
+cyanobyte: 0.1.0
+info:
```
We can delete this file.

Fleker commented on float.emb.yaml:
```diff
@@ -0,0 +1,44 @@
+cyanobyte: 0.1.0
+filename: float.emb
```
We can delete this file.

Fleker commented on float_emb_format.json:
```diff
@@ -0,0 +1,7159 @@
+{
```
We can delete this file.

Fleker commented on template_emboss.h:
```diff
@@ -0,0 +1,621 @@
+// Generated by the Emboss compiler.  DO NOT EDIT!
```
We can delete this file.

chloeyutianyi commented:
@googlebot I signed it!

On Tue, Jul 28, 2020 at 10:24 AM Nick <notifications@github.com> wrote:

> *@Fleker* requested changes on this pull request.
> ------------------------------
>
> In cyanobyte/codegen.py
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/210*discussion_r461745409__;Iw!!LIr3w8kk_Xxm!9nn5MaBrIwXCid_hpzE3xgRxAWKTU0Z1WsdhV5r_Ksx5cG094JuMk24IKMdGn9k$>
> :
>
> >  try:
>      from yaml import CLoader as Loader
>  except ImportError:
>      from yaml import Loader
>  from jinja2 import Environment, FileSystemLoader
>
>  _VERSION = "0.1.0"
> -_DEBUG = False
> +_DEBUG = True
>
> Can we keep this as False. It can be set using the --debug true on the
> command line
> ------------------------------
>
> In cyanobyte/codegen.py
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/210*discussion_r461745782__;Iw!!LIr3w8kk_Xxm!9nn5MaBrIwXCid_hpzE3xgRxAWKTU0Z1WsdhV5r_Ksx5cG094JuMk24IfrGI7i8$>
> :
>
> >          if 'imports' in peripheral_data:
>              for emboss_key in peripheral_data['imports']:
>                  emboss_filename = peripheral_data['imports'][emboss_key]
>                  dir_path = os.path.dirname(os.path.realpath(peripheral))
> +                emboss_filepath = os.path.join(dir_path, emboss_filename)
> +                peripheral_data['imports'][emboss_key] = convert_emb_to_yaml(emboss_filepath)
> +                if _DEBUG:
> +                    print('Imported ' + emboss_filename)
> +                """
>                  with open(os.path.join(dir_path, emboss_filename)) as emboss_file:
>
> Can we remove the commented lines, as we'll not need them.
> ------------------------------
>
> In cyanobyte_float.emb.h
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/210*discussion_r461746696__;Iw!!LIr3w8kk_Xxm!9nn5MaBrIwXCid_hpzE3xgRxAWKTU0Z1WsdhV5r_Ksx5cG094JuMk24I5PDy8SQ$>
> :
>
> > @@ -0,0 +1,1415 @@
> +// Generated by the Emboss compiler.  DO NOT EDIT!
>
> We can delete this file from this commit.
> ------------------------------
>
> In emboss_cyanobyte.py
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/210*discussion_r461746952__;Iw!!LIr3w8kk_Xxm!9nn5MaBrIwXCid_hpzE3xgRxAWKTU0Z1WsdhV5r_Ksx5cG094JuMk24IdEEeCzI$>
> :
>
> > @@ -0,0 +1,86 @@
> +"""Main driver program for using cyanobyte as the backend for emboss"""
>
> We can delete this file from this commit.
> ------------------------------
>
> In emboss_float.emb.h
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/210*discussion_r461747023__;Iw!!LIr3w8kk_Xxm!9nn5MaBrIwXCid_hpzE3xgRxAWKTU0Z1WsdhV5r_Ksx5cG094JuMk24Ib_5oZGs$>
> :
>
> > @@ -0,0 +1,1460 @@
> +// Generated by the Emboss compiler.  DO NOT EDIT!
> +#ifndef TESTDATA_FLOAT_EMB_H_
>
> We can delete this file.
> ------------------------------
>
> In emboss_float.yaml
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/210*discussion_r461747187__;Iw!!LIr3w8kk_Xxm!9nn5MaBrIwXCid_hpzE3xgRxAWKTU0Z1WsdhV5r_Ksx5cG094JuMk24InOizRGI$>
> :
>
> > @@ -0,0 +1,45 @@
> +cyanobyte: 0.1.0
> +info:
>
> We can delete this file.
> ------------------------------
>
> In float.emb.yaml
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/210*discussion_r461747243__;Iw!!LIr3w8kk_Xxm!9nn5MaBrIwXCid_hpzE3xgRxAWKTU0Z1WsdhV5r_Ksx5cG094JuMk24IjB-iyQ8$>
> :
>
> > @@ -0,0 +1,44 @@
> +cyanobyte: 0.1.0
> +filename: float.emb
>
> We can delete this file.
> ------------------------------
>
> In float_emb_format.json
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/210*discussion_r461747382__;Iw!!LIr3w8kk_Xxm!9nn5MaBrIwXCid_hpzE3xgRxAWKTU0Z1WsdhV5r_Ksx5cG094JuMk24ILRWIv7Q$>
> :
>
> > @@ -0,0 +1,7159 @@
> +{
>
> We can delete this file.
> ------------------------------
>
> In template_emboss.h
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/210*discussion_r461747527__;Iw!!LIr3w8kk_Xxm!9nn5MaBrIwXCid_hpzE3xgRxAWKTU0Z1WsdhV5r_Ksx5cG094JuMk24I4TATYnU$>
> :
>
> > @@ -0,0 +1,621 @@
> +// Generated by the Emboss compiler.  DO NOT EDIT!
>
> We can delete this file.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/210*pullrequestreview-456847547__;Iw!!LIr3w8kk_Xxm!9nn5MaBrIwXCid_hpzE3xgRxAWKTU0Z1WsdhV5r_Ksx5cG094JuMk24Iibcdnuo$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AHOX524YG67UAYDUY7KAQBLR54CTFANCNFSM4PJQDUTA__;!!LIr3w8kk_Xxm!9nn5MaBrIwXCid_hpzE3xgRxAWKTU0Z1WsdhV5r_Ksx5cG094JuMk24IKaDPNsA$>
> .
>

chloeyutianyi commented:
@googlebot I fixed it.

On Tue, Jul 28, 2020 at 10:33 AM google-cla[bot] <notifications@github.com>
wrote:

> We found a Contributor License Agreement for you (the sender of this pull
> request), but were unable to find agreements for all the commit author(s)
> or Co-authors. If you authored these, maybe you used a different email
> address in the git commits than was used to sign the CLA (login here
> <https://urldefense.com/v3/__https://cla.developers.google.com/__;!!LIr3w8kk_Xxm!8aNGwf4OiBKcNGxPajzKw7dEQt5pD71wGNqQlccW0e3pwAAzNKJzuaxQtf_R_BM$>
> to double check)? If these were authored by someone else, then they will
> need to sign a CLA as well, and confirm that they're okay with these being
> contributed to Google.
> In order to pass this check, please resolve this problem and then comment @googlebot
> I fixed it.. If the bot doesn't comment, it means it doesn't think
> anything has changed.
>
> ℹ️ *Googlers: Go here
> <https://urldefense.com/v3/__https://goto.google.com/prinfo/https*3A*2F*2Fgithub.com*2Fgoogle*2Fcyanobyte*2Fpull*2F210__;JSUlJSUlJQ!!LIr3w8kk_Xxm!8aNGwf4OiBKcNGxPajzKw7dEQt5pD71wGNqQlccW0e3pwAAzNKJzuaxQo08ZOHY$>
> for more info*.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/google/cyanobyte/pull/210*issuecomment-665175217__;Iw!!LIr3w8kk_Xxm!8aNGwf4OiBKcNGxPajzKw7dEQt5pD71wGNqQlccW0e3pwAAzNKJzuaxQSrSpagg$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AHOX5274W4LHUZOSFHYR4LLR54DWNANCNFSM4PJQDUTA__;!!LIr3w8kk_Xxm!8aNGwf4OiBKcNGxPajzKw7dEQt5pD71wGNqQlccW0e3pwAAzNKJzuaxQD2S01L8$>
> .
>
